### DISCUSSION TITLE
High-quality "Get Add-on" icons

### DISCUSSION THREAD

**Discussion by SecretX33** [FastForwardTeam/FastForward#1159] [General]
I'm wondering if this project would be interested in using high-quality versions of the "Get Add-on" icons for [Chrome](https://user-images.githubusercontent.com/585534/107280622-91a8ea80-6a26-11eb-8d07-77c548b28665.png) and [Firefox](https://user-images.githubusercontent.com/585534/107280546-7b9b2a00-6a26-11eb-8f9f-f95932f4bfec.png).

If you are, I can create a pull request (PR) to update the README.md file with these icons. Specifically, for the Chrome icon, I have both a square version (basically the HQ version of the current one used by the project), and a custom variant with rounded corners, which closely resembles the Firefox icon. I'm curious to know if the project would prefer using this rounded-corner variant, as I find it more visually appealing, personally. 😄

**Comment by NotAProton** [comment]
Can you upload the both of them?

**Comment by NotAProton** [comment]
-> #1160
